### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by NadyaKir**
Added functionality: checkbox change card styles.

### CODE REVIEW

**Comment by Jaskon** [comment]
In this PR you have changes from task-2 and task-1. Could you please only show here task-2 changes?
(You need to merge task-1 into task-2)

**Comment by NadyaKir** [comment]
> In this PR you have changes from task-2 and task-1. Could you please only show here task-2 changes? (You need to merge task-1 into task-2)

Done

File: src/components/Card/Card.js [hunk 1]
```diff
@@ -1,10 +1,30 @@
+import { useState } from 'react';
+
 import './Card.css';
+
+//god help push this branch
```
Comment (Jaskon) [inline src/components/Card/Card.js]: Remove this please :)

**Review by Jaskon — CHANGES_REQUESTED** [review]

File: src/components/Card/Card.js (hunk 1 above)
Comment (NadyaKir) [reply src/components/Card/Card.js]: Okay))

**Review by Jaskon — APPROVED** [review]

**Pull request merged**